packages/react/src/slider/root/SliderRoot.tsx

```diff
@@ -77,14 +79,13 @@ export const SliderRoot = React.forwardRef(function SliderRoot<
   const onValueChange = useEventCallback(
     onValueChangeProp as (
       value: number | number[],
-      data: BaseUIEventDetails<'none'>,
-      activeThumbIndex: number,
+      data: BaseUIChangeEventDetails<'none', { activeThumbIndex: number }>,
```

Comment:
Wouldn't this work here? 🤔 
```suggestion
      data: ChangeEventDetails,
```